docs/jax.debug.rst

```diff
@@ -6,14 +6,28 @@ jax.debug package
 
 .. automodule:: jax.debug
 
-Debugging utilities
---------------------------
+Runtime value debugging utilities
+---------------------------------
 
-:doc:`debugging/print_breakpoint` describes how to make use of JAX's debugging features.
+:doc:`debugging/print_breakpoint` describes how to make use of JAX's runtime value
+debugging features.
 
 .. autosummary::
   :toctree: _autosummary
 
   callback
   print
   breakpoint
+
+Sharding debugging utilities
+----------------------------
+
+Functions that enable inspecting and visualizing array shardings inside (and outside)
+staged functions.
+
+.. autosummary::
+  :toctree: _autosummary
+
+  inspect_array_sharding
+  visualize_array_sharding
```

Comment:
Why not just expose `visualize_sharding`?

Reply:
It can't be used inside of `pjit` whereas `visualize_array_sharding` can be.

Reply:
Nevermind, I guessed wrong about what these things do.